### ISSUE TITLE
require(...) is not a function

### ISSUE THREAD

**Issue by RacknerFrank** [ConvertAPI/convertapi-node#37]
I've been trying to update convertapi from 1.5.0 to 1.10.1 but have been having trouble with it. I am specifically attempting to update to 1.10.1 as it solves an earlier issue I had opened https://github.com/ConvertAPI/convertapi-node/issues/36

My code is: 
```
const convertapi = require('convertapi')(process.env.CONVERT_API_KEY);
```

My package.json line is:
```
"convertapi": "^1.10.1"
```

and my package.lock has:
```
    "convertapi": {
      "version": "1.10.1",
      "resolved": "https://registry.npmjs.org/convertapi/-/convertapi-1.10.1.tgz",
      "integrity": "sha512-lReLTolrMC+hbEmcY2MjSsqK9zf2YHnjZLrQMd9HK4dvLwLFs5jzvG42Ps8MyT8qzV8AHIMZNlgoPG2t6AzVtw==",
      "requires": {
        "axios": "^0.21.1"
      }
    },
```

This code throws the error:
```
TypeError: require(...) is not a function\n    at /path/to/file.js:145:66
```
This was not happening in 1.5.0

*I went through each version one by one and it worked up until 1.9.1 and stopped working for 1.10.0

**Comment by RacknerFrank** [comment]
I took another look at this and got it work by changing the syntax to: 
```
const ConvertAPI = require('convertapi').ConvertAPI;
const convertapi = new ConvertAPI(process.env.CONVERT_API_KEY);
```

**Comment by laurynas-baltsoft** [comment]
Hi @RacknerFrank, thank you for the reporting this issue.
Fixed it and released `1.11.0` version.
